src/javascript/_common/base/gtm.js

```diff
@@ -52,6 +52,7 @@ const GTM = (() => {
             pushDataLayer({ bom_affiliate_token: affiliate_token.t });
         }
 
+        const now = moment(window.time || Date.now()).unix().utc();
```

Comment:
I suggest `moment(window.time || undefined)` instead of using a combination of Date and moment

Reply:
I don't think we can turn unix value to utc, it should be first utc and then unix if you want a unix value in the end. if so, perhaps it's better to rename the variable to make it clear what value now is supposed to be, something like `moment_now` or `unix_now`